tests/h/subscribers_test.py

```diff
@@ -7,14 +7,6 @@
 from h.exceptions import RealtimeMessageQueueError
 
 
-class FakeMailer:
-    def __init__(self):
-        self.lastcall = None
-
-    def __call__(self, recipients, subject, body, html):
-        self.lastcall = (recipients, subject, body, html)
```

Comment:
Yeah, we don't really want to have fakes like this. Again - old code. Or someone who thought that "mocks are bad" because they don't really test things (I've seen people actually give that as a reason for writing their own fake class like this)